Inseok_45/controllers/userController.js

```diff
@@ -20,4 +25,28 @@ const signUp = async ( req, res ) => {
     };
 };
 
-module.exports = { signUp };
\ No newline at end of file
+const login = async (req, res) => {
+    try{
+        const { userName, password } = req.body;
+
+        if(!userName){
+            return res.status(400).json({ message: "Username Required" });
+        } else if(!password){
+            return res.status(400).json({ message: "Password Required" });
+        }
+        
+        let result = await userService.login( userName, password );
+
+        if(result === true){
+            return res.status(200).json({ accessToken: jwtToken });
+        } else {
+            return res.status(400).json({ message: "Invalid User" })
+        }
```

Comment:
```suggestion
        if(!result) return res.status(400).json({ message: "Invalid User" })
        
        return res.status(200).json({ accessToken: jwtToken });
```
로직은 성공 케이스를 조건문으로 작성하는것보다, 모든 로직에 대해서 실패 분기 조건들을 검증한 뒤 최종적으로 성공을 return하는 식으로 작성되는게 좋습니다. 또한 else는 가급적 쓰지 않도록 해주세요! 가독성을 해치며 대부분의 경우에는 else 없이도 결국 원하는 로직으로 작성이 가능하기 때문입니다.